content/en/blog/_posts/2024-09-21-cncf-dhhwg-spotlight/index.md

```diff
@@ -0,0 +1,78 @@
+
+## Impact and the role of allies
+**SK: Can you share some key impacts/outcomes of the conference best practices document?**
+
+**CP**: Here are the two most important ones: Captions should be on the monitor, not in an app. That's especially important during technical talks with live demos. Deaf and hard of hearing attendees will miss important information switching between captions on their phone and code on the screen. 
+
+Interpreters are most valuable during networking, not in talks (with captions). Most people come to conferences for the hallway track. That is no different for deaf attendees. If they can't network, they are missing out on key professional connections, affecting their career prospects. 
+
+
+**SK: In your view, how crucial is the role of allies within the DHHWG, and what contributions have they made to the group’s success?**
+
+**CP**: Deaf and hard of hearing individuals are a minority and can only do so much. ***Allies are key to any diversity and inclusion initiative***. As a majority, allies can help spread the word and educate their peers, playing a key role in scaling advocacy efforts. They also have the power to demand change. It's easy for companies to ignore minorities, but if the majority demands that their employers be accessible, environmentally conscious, and good citizens, they will ultimately be pushed to adapt to new societal values. 
+
+## Expanding DEI efforts and future vision
+**SK: The importance of allies in driving change is clear. Beyond the DHHWG, are you involved in any other DEI groups or initiatives within the tech community?**
+
+**CP**: As mentioned above, I'm working on an initiative called ABIDE, which is still work in progress. I don't want to share too much about it yet, but what I can say is that the DHHWG will be part of it and that we just started a Blind and Visually Impaired WG (BVIWG). ABIDE will start by focusing on accessibility, so if anyone reading this has an idea for another WG, please reach out to me via the CNCF Slack @Catherine Paganini.
+
+**SK: What does the future hold for the DHHWG? Can you share details about any ongoing or upcoming initiatives?**
+
+**CP**: I think we've been very successful in terms of visibility and awareness so far. We can't stop, though. Awareness work is ongoing, and most people in our community haven't heard about us or met anyone on our team yet, so a lot of work still lies ahead. 
+![DHHWG members at KubeCon Paris](cncf-dhhwg-paris.jpg)
```

Comment:
Line breaks are needed before and after the image link so this doesn't get rendered all together.

Reply:
Thanks @stmcginnis . Incorporated your suggestion and added line breaks before and after both the image links.